core/tests/puppeteer-acceptance-tests/utilities/user/logged-out-user.ts

```diff
@@ -951,6 +1007,198 @@ export class LoggedOutUser extends BaseUser {
+   * The button is in the bottom section of the page.
+   */
+  async clickPartnerWithUsButtonInPartnershipsPageInGivenLanguage(
+    langCode: string
+  ): Promise<void> {
+    await this._changeSiteLanguage(langCode);
+
+    await this.clickButtonToNavigateToNewPage(
+      partnerWithUsButtonAtTheBottomOfPartnershipsPage,
+      'Partner With Us button at the bottom of the Partnerships page',
+      partnershipsFormInPortugueseUrl,
+      'Partnerships Google Form'
+    );
+  }
+
+  /**
+   * Function to click the Download Brochure button in the Partnerships page
+   * and check if it opens the Partnerships Brochure.
+   */
+  async clickDownloadBrochureButtonInPartnershipsPage(): Promise<void> {
+    await this.openExternalPdfLink(
+      brochureButtonInPartnershipsPage,
+      partnershipsBrochureUrl
+    );
+  }
+
+  /**
+   * Function to click the first Read blog post  in the Partnerships page
+   * and check if it opens the blog page.
+   */
+  async clickReadBlogPostLinkInPartnershipsPage(): Promise<void> {
+    await this.page.waitForXPath('//a[contains(text(),"Read blog post")]');
+    await this.clickButtonToNavigateToNewPage(
+      readBlogPostButtonInPartnershipsPage,
+      'Read blog post button',
+      blogPostUrlinPartnershipsPage,
+      'Blog Post'
+    );
+  }
+
+  /**
+   * Function to click the Read more stories button in the Partnerships page
+   * and check if it opens the blog page.
+   */
+  async clickReadMoreStoriesButtonInPartnershipsPage(): Promise<void> {
+    await this.clickButtonToNavigateToNewPage(
+      readMoreStoriesButtonInPartnershipsPage,
+      'Read more stories button',
+      blogUrl,
+      'Blog'
+    );
+  }
+
+  /**
+   * Function to click the Apply To Volunteer at the top of the Volunteer page
+   * and check if it opens the Volunteer form.
+   */
+  async clickApplyToVolunteerAtTheTopOfVolunteerPage(): Promise<void> {
+    await this.page.waitForSelector(
+      applyToVolunteerButtonAtTheTopOfVolunteerPage
+    );
+    await this.page.click(applyToVolunteerButtonAtTheTopOfVolunteerPage);
+
+    await this.page.waitForTimeout(100);
+    const pages = await this.page.browser().pages();
+    const newTab = pages[pages.length - 1];
+    await newTab.waitForNavigation();
+
+    if (this.page.url() !== volunteerFormUrl) {
+      throw new Error(
+        'The volunteer button did not navigate to the expected URL.'
+      );
+    }
+  }
+
+  /**
+   * Function to click the Apply To Volunteer at the bottom of the Volunteer page
+   * and check if it opens the Volunteer form.
+   */
+  async clickApplyToVolunteerAtTheBottomOfVolunteerPage(): Promise<void> {
+    await this.page.waitForSelector(
+      applyToVolunteerButtonAtTheBottomOfVolunteerPage
+    );
+    await this.page.click(applyToVolunteerButtonAtTheBottomOfVolunteerPage);
+
+    await this.page.waitForTimeout(100);
+    const pages = await this.page.browser().pages();
+    const newTab = pages[pages.length - 1];
```

Comment:
It's ok. However, have you figured out why the clickButtonToNavigateToNewPage() ain't working in this case?